active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,271 @@
+must be grouped, which is done with the Xaml [RadioButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.RadioButton)
+control by setting the GroupName property on each button to the same value.
+
+That mechanism is awkward, and doesn't by default give the right keyboarding or
+accessibility behavior.
+
+The RadioButtons control in this spec is introduced to help solve those issues.
+It's like a ListView control, but the items in the list are RadioButton controls.
+
+# Description
+
+RadioButtons is a new control that enables you to create groups of RadioButton elements easily,
+while also correctly supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective group of RadioButton elements
+using the RadioButtons control.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Item 1</x:String>
+    <x:String>Item 2</x:String>
+    <x:String>Item 3</x:String>         
+</muxc:RadioButtons>
+```
+
+![alt text](DefaultRadioButtonGroup.png)
+
+## DataBinding with RadioButtons
+An example of simple data binding of the ItemsSource with RadioButtons.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" ItemsSource="{x:Bind radioButtonItems}" />
+```
+
+```C#
+public sealed partial class MainPage : Page
+{
+    public class OptionDataModel
+    {
+        public string Label;
+        public override string ToString()
+        {
+            return Label;
+        }
+    }
+
+    List<OptionDataModel> radioButtonItems;
+
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        radioButtonItems = new List<OptionDataModel>();
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 1" });
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 2" });
+        radioButtonItems.Add(new OptionDataModel() { Label = "Item 3" });
+    }
+}
+```
+
+## Vertical and horizontal RadioButton lists
+In instances where there is more that one column of RadioButton items within a single RadioButtons list control, the list will flow a column-major layout.
+
+|MaxColumns not specified <br>OR<br> MaxColumns = 1 |  MaxColumns = 4 |
+|:--:|:--:|
+| ![alt text](singlecolumn_example.png) | ![alt text](multicolumn_example_1.png) |
+
+```xml
+<RadioButtons Header="Select Number" MaxColumns="2">
+    <x:String>1</x:String>
+    <x:String>2</x:String>
+    <x:String>3</x:String>
+    <x:String>4</x:String>
+</RadioButtons>
+```
+
+By default, the RadioButtons list will fill itself vertically, meaning if no ``MaxColumns`` value is set, it will be assumed to be 1 and orient itself vertically.
+
+### Multiple columns (uncommon case)
+
+If you specify a ``MaxColumns`` value that is not equal to the number of items in the RadioButtons list, the list will arrange  itself in column-major order and put any uneven/remaining items in the first column, in the case of an uneven ``MaxColumns`` value or list items defined.
```

Comment:
I was following along but got lost in this sentence by the time I got to "in the case of an uneven ``MaxColumns`` value or list items defined." 😅